applications/FluidDynamicsApplication/custom_python/add_custom_utilities_to_python.cpp

```diff
@@ -15,6 +15,9 @@
 // System includes
 
 // External includes
+#ifdef KRATOS_USE_AMATRIX
+#include "boost/numeric/ublas/matrix.hpp" // for the sparse space dense vector
```

Comment:
This is a temporary solution until @pooyan-dadvand implements the sparse matrix in AMatrix. Am I wrong?

Reply:
Yes, it is needed now to keep using the ublas sparse vectors. Everything wrapped in `KRATOS_USE_AMATRIX` precompiler checks should be removed eventually.